firstvoices/backend/serializers/song_serializers.py

```diff
@@ -0,0 +1,47 @@
+from rest_framework.serializers import ModelSerializer
+
+from backend.models import Song, TranslatedText
+from backend.serializers.base_serializers import SiteContentLinkedTitleSerializer, base_timestamp_fields
+from backend.serializers.media_serializers import RelatedMediaSerializerMixin, ImageSerializer
+from backend.serializers.site_serializers import LinkedSiteSerializer
+
+
+class TranslatedTextSerializer(ModelSerializer):
+    class Meta:
+        model = TranslatedText
+        fields = ('text', 'language')
+
+
+class SongDetailSerializer(
+    RelatedMediaSerializerMixin,
+    SiteContentLinkedTitleSerializer
+):
+    cover_image = ImageSerializer()
+    site = LinkedSiteSerializer()
+    title_translations = TranslatedTextSerializer(many=True)
+    introduction_translations = TranslatedTextSerializer(many=True)
+    lyrics_translations = TranslatedTextSerializer(many=True)
+
+    class Meta(SiteContentLinkedTitleSerializer.Meta):
+        model = Song
+        fields = (
+            base_timestamp_fields
+            + RelatedMediaSerializerMixin.Meta.fields
+            + ("url",
+               "id",
+               "title",
+               "site",
+               "cover_image",
+               "title",
+               "title_translations",
+               "introduction",
+               "introduction_translations",
+               "lyrics",
+               "lyrics_translations",
```

Comment:
@gmcauliffe I think we'll need your perspective on how to best return translated text data. I wonder if one list of objects would be easier to use than two lists that have to be matched up? Something like 

`lyrics: [ { text: xyz, translations: [ { text: xyz, language: abc }] }]`

Is that overkill?

Reply:
Maybe also @caforbes ?

Reply:
One list would be great. It would be great if we could simplify it even further by dropping the translation language and making it a one to one list of text and translation. Say:
`lyrics: [
    { text: 'Liimin ‘wii sgusiit', translation: 'Sing you big potato' }, 
    { text: 'Amii dm ‘wah liimin.', translation: 'If you do not sing' },
 ]`

Reply:
@gmcauliffe Does this mean we assume all translations for lyrics are in English? Or should there be different lists for different languages? eg `lyrics : { EN: [ {text, translation},  {text, translation}], FR: [ {text, translation}, {text, translation} ]}`.

Reply:
That's right, we will assume all translations for a language site will be in the same language. Thanks for checking.